caim_base/templates/comments/delete_reply.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'base/wrapper.html' %}
+{% load avatar_tags %}
+
+{% block content %}
+<div class="container">
+    <div class="row justify-content-md-center">
+      <div class="col-md-8">
+        <div class="container py-5">
+            <h3>Are you sure you want to delete this reply?</h3>
+
+            <br />
+
+            <div class="comment" id="comment-{{comment.id}}">
+                {% avatar comment.user 40 40 class="comment-avatar" %}
```

Comment:
Shoudl this be animalsubcomment not comment?

Reply:
fixed